proposed/2023/Add-NoHttpCacheOption.md

```diff
@@ -0,0 +1,61 @@
+# Title
+
+- Author Name (https://github.com/Nigusu-Allehu)
+- Start Date (YYYY-MM-DD)
+- GitHub Issue (https://github.com/NuGet/Home/issues/9180)
+- GitHub PR (GitHub PR link)
+
+## Summary
+
+<!-- One-paragraph description of the proposal. -->
+The proposal seeks to eliminate user confusion regarding the `-NoCache`, `--no-cache`, and `RestoreNoCache` CLI options in dotnet.exe, nuget.exe, and msbuild.exe by introducing a new CLI option, `--no-http-cache`, which clearly specifies that it disables only HTTP cache caching, not the global packages folder. To ensure a smooth transition and avoid additional confusion, the old options will not be deprecated but will co-exist with the new option. Updated documentation will provide guidance on the specific functions of each option, allowing users to make informed decisions based on their caching needs.
```

Comment:
if we're gonna list all options, we should list them in the correct order and say, `respectively` at the end of that sentence. 

Note that these are not *all* CLI options, as `RestoreNoCache` is a property. I'd just call them options to be unambiguously correct.